### PR TITLE
Added Tailwind CSS

### PR DESCRIPTION
**Pull request by TheInfinityIQ**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -4,20 +4,10 @@ import HelloWorld from './components/HelloWorld.vue'
...
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-    </div>
-  </header>
-
-  <RouterView />
+  <div class="container mx-auto bg-gray-200 rounded-xl shadow border p-8 m-10">
```
Comment (Dragwar) [inline src/App.vue:7]: from this
image

this to
image

```suggestion
  <div class="container mx-auto rounded-xl shadow border p-8 m-10">
```

File: src/main.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import './assets/main.css'
+import './style.css'
```
Comment (Dragwar) [inline src/main.ts:2]: Why not move import the tailwind stuff in `main.css`? (instead of creating a whole other css file)

File: src/style.css
Comment (Dragwar) [inline src/style.css:1]: Why not move this to `src/assets/main.css` or `src/assets/base.css`?

**Review by Dragwar — CHANGES_REQUESTED** [review]
left some small change requests (small CSS clean up and dummy visual improvement)

**Comment by TheInfinityIQ** [comment]
Thank you for fixing the sillies! I was copying tutorial 100% even when creating unneeded files just to make sure it worked seeing that this is my first time working with TailwindCSS.

**Pull request merged**